src/init.cpp

```diff
@@ -1118,13 +1118,35 @@ bool AppInit2(boost::thread_group& threadGroup)
     }
 
     if (!fTxIndex) {
-        return InitError(_(
-                "Disabled transaction index detected.\n\n"
-                "Omni Core requires an enabled transaction index. To enable "
-                "transaction indexing, please use the \"-txindex\" option as "
-                "command line argument or add \"txindex=1\" to your client "
-                "configuration file."
-            ));
+        // ask the user if they would like us to modify their config file for them
+        std::string msg = _("Disabled transaction index detected.\n\n"
+                            "Omni Core requires an enabled transaction index. To enable "
+                            "transaction indexing, please use the \"-txindex\" option as "
+                            "command line argument or add \"txindex=1\" to your client "
+                            "configuration file within your data directory.\n\n"
+                            "Configuration file: "); // allow translation of main text body while still allowing differing config file string
```

Comment:
If you look at any of the localization files, this is what I see (& hence thought whole strings were translated):
```
<message>
        <source>Double-click to edit address or label</source>
        <translation>Для того, чтобы изменить адрес или метку, дважды кликните по изменяемому объекту</translation>
    </message>
```